store/todos.js

```diff
@@ -30,9 +33,16 @@ export const mutations = {
       createdAt: new Date(),
       completedAt: null,
     }
-
     state.list = [newTodo, ...state.list]
   },
+
+  update(state, updatedTodo) {
+    const todoIndex = state.list.findIndex((t) => t.id === updatedTodo.id)
+    const todo = state.list[todoIndex]
+    todo.title = updatedTodo.title
+    state.list.splice(todoIndex, 1, todo)
+  },
```

Comment:
nitpicking: we could use `map` here for cleaner syntax.

Reply:
resolved